Library/Homebrew/cmd/install.rb

```diff
@@ -310,9 +310,7 @@ def run
         Install.perform_preinstall_checks_once
         Install.check_cc_argv(args.cc)
 
-        Install.ask_formulae(installed_formulae, args: args) if args.ask?
-
-        Install.install_formulae(
+        formulae_installer = Install.get_formulae_dependencies(
```

Comment:
Drop `get_*` from all of these new methods; that's not how they are typically named in Ruby.